src/domain/kaivuilmoitus/KaivuilmoitusContainer.tsx

```diff
@@ -173,6 +200,8 @@ export default function KaivuilmoitusContainer({ hankeData, application }: Reado
         formSteps={formSteps}
         formData={watchFormValues}
         onStepChange={handleStepChange}
+        isLoading={attachmentsUploading}
+        isLoadingText={attachmentsUploadingText}
       >
         {function renderFormActions(activeStepIndex, handlePrevious, handleNext) {
           async function handleSaveAndQuit() {
```

Comment:
Form buttons (next, previous, etc.) are missing loading indicators when attachments are uploading.

Reply:
I was not careful enough when trying to copy-paste things from Johtoselvitys form container. Thank you for noticing!

Reply:
Works as expected now.